go/teams/team_loader.go

```diff
@@ -0,0 +1,19 @@
+package teams
```

Comment:
I'm unsure on where to go from here. How exactly does KBFS make the RPC call? is it enough to just provide this function? Also unclear on who provides ctx and g and what they're responsible for.